Content.Server/Commands/ActionCommands.cs

```diff
@@ -31,13 +31,13 @@ public void Execute(IConsoleShell shell, string argStr, string[] args)
         var actionUpgrade = _entMan.EntitySysManager.GetEntitySystem<ActionUpgradeSystem>();
         var id = args[0];
 
-        if (!NetEntity.TryParse(id, out var nuid))
+        if (!EntityUid.TryParse(id, out var uid))
```

Comment:
Commands should be using netentities.

Reply:
Oop I just copied over the entire fifle, didn't see the netent changes.